supervisor-services-labs/argocd-operator/argocd-operator.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  creationTimestamp: null
+  name: argocd-operator.fling.vsphere.vmware.com
+spec:
+  categories:
+  - CI/CD
+  - GitOps
+  displayName: argocd-operator
+  iconSVGBase64: none
+  longDescription: The Argo CD Operator manages the full lifecycle for Argo CD and
+    its components. The operator's goal is to automate the tasks required when operating
+    an Argo CD cluster. Beyond installation, the operator helps to automate the process
+    of upgrading, backing up and restoring as needed and remove the human as much
+    as possible. In addition, the operator aims to provide deep insights into the
+    Argo CD environment by configuring Prometheus and Grafana to aggregate, visualize
+    and expose the metrics already exported by Argo CD.
+  maintainers:
+  - name: vSphere IaaS and Consumption
+  providerName: VMware
+  shortDescription: The Argo CD Operator manages the full lifecycle for Argo CD and
+    its components.
+---
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  creationTimestamp: null
+  name: argocd-operator.fling.vsphere.vmware.com.0.8.0
+spec:
+  refName: argocd-operator.fling.vsphere.vmware.com
+  releasedAt: "2024-03-25T18:23:27Z"
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/vsphere-labs/argocd-operator@sha256:9d725c3bfd7207746c307af3223e040e85647bb4f532a6716dcf6df8fa61805e
```

Comment:
Note: can we document internally how to manage that package?

Reply:
Yes. This will be done and shared as a Confluence page.